README.md

```diff
@@ -390,6 +390,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Zuplo](https://zuplo.com/) - Free API Management platform to design, build, and deploy APIs to the Edge. Add API Key authentication, rate limiting, developer documentation and Monetization to any API in minutes. OpenAPI-native and fully-programmable with web standard apis & Typescript. The free plan offers up to 10 projects, unlimited production edge environments, 250 API keys, 100K monthly requests, and 1GB egress.
   * [Amazon SageMaker](https://studiolab.sagemaker.aws/) — A cloud-based ML platform with JupyterLab and terminal access. The free tier includes 8 hours of CPU (max 4 hours per session) or 4 hours of GPU per day.
   * [FormatJSONOnline.com](https://formatjsononline.com) - A free, browser-based tool to format, validate,compare and minify JSON data instantly.
+  * [Orshot](https://orshot.com) - Create and automate marketing assets at scale. Generate dynamic images from templates with Rest API, SDKs, Zapier and other integrations. 500 Renders free, all features included.
```

Comment:
sorry, 500 renders free one off is not a free tier its like a trial so wouldn't be acceptable.

A free tier is free every month.